.github/workflows/renovate.yml

```
@@ -0,0 +1,31 @@
+name: Renovate
+on:
+  # Allows manual/automated trigger for debugging purposes
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: "Override default log level"
+        required: false
+        default: "info"
+        type: string
+      overrideSchedule:
+        description: "Override all schedules"
+        required: false
+        default: "false"
+        type: string
+  schedule:
+    - cron: '0 8 * * *'
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
```

We should add this to avoid running and failing on forks:
```suggestion
    runs-on: ubuntu-latest
    if: |
      !github.event.repository.fork && 
      !github.event.pull_request.head.repo.fork
```